__tests__/app.test.js

```diff
@@ -71,3 +71,29 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+describe("/api/articles", () => {
+  test("GET:200, responds with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(13);
+        body.articles.forEach((article) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("string");
+          expect(article.body).toBe(undefined);
```

Comment:
Great test here! Great to see you are checking the length before your forEach.
You could be more explicit and check that it doesn't have the body property. Like this:
`expect(article.body).not.toHaveProperty("body")`